instrumentation/pekko/pekko-http-1.0/javaagent/build.gradle.kts

```diff
@@ -10,28 +10,57 @@ muzzle {
     versions.set("[1.0,)")
     assertInverse.set(true)
     extraDependency("org.apache.pekko:pekko-stream_2.12:1.0.1")
+    extraDependency("com.softwaremill.sttp.tapir:tapir-server_2.12:1.7.0")
+    extraDependency("com.softwaremill.sttp.tapir:tapir-core_2.12:1.7.0")
   }
   pass {
     group.set("org.apache.pekko")
     module.set("pekko-http_2.13")
     versions.set("[1.0,)")
     assertInverse.set(true)
     extraDependency("org.apache.pekko:pekko-stream_2.13:1.0.1")
+    extraDependency("com.softwaremill.sttp.tapir:tapir-server_2.13:1.7.0")
+    extraDependency("com.softwaremill.sttp.tapir:tapir-core_2.13:1.7.0")
   }
   pass {
     group.set("org.apache.pekko")
     module.set("pekko-http_3")
     versions.set("[1.0,)")
     assertInverse.set(true)
     extraDependency("org.apache.pekko:pekko-stream_3:1.0.1")
+    extraDependency("com.softwaremill.sttp.tapir:tapir-server_3:1.7.0")
+    extraDependency("com.softwaremill.sttp.tapir:tapir-core_3:1.7.0")
+  }
+  pass {
+    group.set("com.softwaremill.sttp.tapir_2.12")
+    module.set("tapir-pekko-http-server")
+    versions.set("[1.7,)")
+    assertInverse.set(true)
+    extraDependency("com.softwaremill.sttp.tapir:tapir-server_2.12:1.7.0")
+    extraDependency("com.softwaremill.sttp.tapir:tapir-core_2.12:1.7.0")
+  }
```

Comment:
[for reviewer] Though I've added the muzzle checks for Tapir here, when I run the `muzzle` gradle command for this module, I don't see muzzle checks for these.